sparse/usr/lib/systemd/system/opt2.mount

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Mount for /opt2
+Requires=opt2-format.service
+After=opt2-format.service
+ConditionPathExists=!/var/cache/appsupport/opt2-no-mount
```

Comment:
/var/cache is for application cache, it has to be able to be savely removed. /etc would make more sense.

https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/var.html

Reply:
Var cache can be deleted and it doesn't do any harm in this implementation, the boot just takes ~100ms longer in worst case (when the partition is used as LVM PV -> meaning it is ignored). It is used just as a cache, and afaik var/cache is supposed to be persistent over reboots, so it serves this purpose fine.